### DISCUSSION TITLE
Reviewer 편하게 추가하는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by wooyaggo86** [ThinkAboutSoftware/AcademicConference#101] [General]
이번에는 일일이 한분씩 아이디를 검색해서 넣었어요.
혹시 더 간편히 추가하는 방법이 있을까요?

**Comment by jongfeel** [comment]
제가 #88 review 하면서 코멘트 달았던 부분이 있는데, 새로 issue로 팠습니다
Github cli를 공식으로 지원하는데 써보시면 좋습니다!

- #93

**Comment by wooyaggo86** [comment]
오! 이런 방법이 있었네요👍 소개해주셔서 감사해요. 다음 챕터에서 활용해 보겠습니다🙏
